WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -9,6 +9,8 @@
 #import "WordPress-Swift.h"
 #import "PhotonImageURLHelper.h"
 
+CGFloat const ReaderPostServiceRemotOffsetRankCap = 1000000.0;
```

Comment:
Actually, I think I'll drop the constant completely and [adopt the approach](https://github.com/wordpress-mobile/WordPress-Android/pull/4500) @nbradbury used in the android app. Its a more elegant solution. :)